keyboards/keebwerk/nano_slider/readme.md

```diff
@@ -0,0 +1,15 @@
+# nano. slider
+
+![nano. slider](https://i.imgur.com/0UxyfxL.png)
+
+A small eight key macropad with slider.
+
+* Keyboard Maintainer: [Duckle](https://github.com/yourusername)
+* Hardware Supported: The PCBs, controllers supported
+* Hardware Availability: Links to where you can find this hardware
```

Comment:
```suggestion
* Keyboard Maintainer: [Duckle29](https://github.com/Duckle29)
* Hardware Supported: nano. slider PCB, ATmega32U4
* Hardware Availability: [keebwerk.](https://www.keebwerk.com/product/nano-slider-keyboard/)
```

Reply:
Man I must've been tired when doing the readme